ticdc/ticdc-changefeed-config.md

```diff
@@ -69,7 +69,7 @@ enable-old-value = true
 
 # Filter rules.
 # Filter syntax: <https://docs.pingcap.com/tidb/stable/table-filter#syntax>.
-rules = ['*.*', '!test.*']
+rules = ['*.*When the downstream is a storage service, the protocol can only be set to canal-json or csv.', '!test.*']
```

Comment:
Is this a typo?

Reply:
Fixed in [68fb1b4](https://github.com/pingcap/docs/pull/10998/commits/68fb1b4b4fa1b6d9d27d1b9c0250d3d5a3b5eefc)